core/parser/prop.go

```diff
@@ -0,0 +1,23 @@
+package prop
+
+import (
+	"fmt"
+)
+
+
+type Prop struct{
+	prop map["string"][]string
+}
+
+
+func Validate(properties *Properties) (int, error) {
```

Comment:
Needs a method comment.

Also, I think you can just return a bool here -- not sure we need (int which seems to be taking the place of bool) or error (the descriptive message doesn't seem terribly helpful)